src/components/Modal/Modal.tsx

```diff
@@ -69,7 +69,7 @@ const Modal: React.FC<ModalProps> = ({
                   <Card minWidth="400px" position="relative" boxShadow="dark200">
                     {title && (
                       <Box as="header" borderBottom="1px solid" borderColor="navyblue-300" py={6}>
-                        <Heading as="h4" size="x-small" textAlign="center" id={id}>
+                        <Heading as={Heading} size="x-small" textAlign="center" id={id}>
```

Comment:
![image](https://user-images.githubusercontent.com/20060118/150617052-f6c9f2d5-90e2-4328-b6bf-554308662b26.png)

Reply:
Heading as heading :P

Reply:
😂 

We might even be able to remove this prop entirely now...I'll give it a try

Reply:
`<Heading as={Heading} ` seems completely wrong 😄  Can we not remove the `as`? It would render an `h2`, right?

Reply:
Yeah this was leftover from one of the previous versions, removed it 😂